src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -48,10 +60,6 @@ const testingTimesheetResp = {
     "Comments":[]
 }
 
-//Example entry in VC jsons: 
-const example_vs_entry = [
-    {row_id:"<Row UUID>", col_key:"StartTime", value:"<A different epoch>"}
-]
 
 //To test uploading a timesheet 
 // apiClient.updateUserTimesheet(testingTimesheetResp); 
```

Comment:
🏕️ Can we extract this into an onboarding doc and remove from the code? Can wait, not blocking

Reply:
ill create ticket to create onboarding material / documentation